sample-app/src/components/Dropdown.tsx

```diff
@@ -23,22 +29,41 @@ interface Props {
 export default function Dropdown({
   options,
   onClickOption = () => {},
+  focusedSectionIndex,
   focusedOptionIndex,
   optionIdPrefix,
   cssClasses
 }: Props): JSX.Element | null {
-  function renderOption(option: Option, index: number) {
+  function renderSection(section: {results: Option[], label?: string}, sectionIndex: number) {
+    return (
+      <div
+        className={cssClasses.optionSection}
+        key={`section-${sectionIndex}`}>
+        {section.label &&
+          <div
+            className={cssClasses.sectionLabel}
+            key={section.label}>
```

Comment:
Should we have `sectionIndex` as part of the key as well in case two sections have the same label? It's unlikely, but adding more to the key shouldn't hurt

Reply:
yes, I'll add that

Reply:
actually, the key is not needed here. that was just from a previous version, so I'll remove it entirely